### ISSUE TITLE
Can't override ResolverTypeWrapper with mappers

### ISSUE THREAD

**Issue by ellipticaldoor** [dotansimha/graphql-code-generator#2236]
**Describe the bug**
 I want to override ResolverTypeWrapper.

By default looks like this:
``` typescript
export type ResolverTypeWrapper<T> = Promise<T> | T;
```

But I want it like this:
``` typescript
export type ResolverTypeWrapper<T> = Promise<DeepPartial<T>> | DeepPartial<T>;
```

**To Reproduce**
If you try this on the **codegen.yml** to override ResolverTypeWrapper it will not work

``` yaml
generates:
  src/generated/apollo/index.ts:
    plugins:
      - "typescript"
      - "typescript-resolvers"
    config:
      mappers:
        ResolverTypeWrapper: Promise<DeepPartial<T>> | DeepPartial<T>
```

I am doing something wrong?

**Comment by ellipticaldoor** [comment]
For the moment I made a custom script with node that I run after codegen.

I will leave it here in case someone needs it.

``` javascript
const path = require('path');
const fs = require('fs');
const util = require('util');
const readFile = util.promisify(fs.readFile);
const writeFile = util.promisify(fs.writeFile);

const currentPath = path.resolve(__dirname, '..');
const schemaFilePath = `${currentPath}/src/server/graphql.ts`;

async function main() {
  let schema = await readFile(schemaFilePath, 'utf8');

  schema = "import { DeepPartial } from 'ts-essentials';\n" + schema;

  schema = schema.replace(
    'export type ResolverTypeWrapper<T> = Promise<T> | T;',
    'export type ResolverTypeWrapper<T> = Promise<DeepPartial<T>> | DeepPartial<T>;',
  );

  await writeFile(schemaFilePath, schema, 'utf8');
}

try {
  main();
} catch (error) {
  console.error(error);
}
```

**Comment by dotansimha** [comment]
Hi @ellipticaldoor ! 
The `mappers` are there only for GraphQL types, and not for output TypeScript types. 
There it no option to override `ResolverTypeWrapper` at the moment. I added it in: https://github.com/dotansimha/graphql-code-generator/pull/2254

**Comment by ellipticaldoor** [comment]
Wow! that's amazing!! Thank you very much @dotansimha

**Comment by dotansimha** [comment]
Available in 1.5.0 🎉
